narrative_llm_agent/workflow_graph/nodes_hitl.py

```diff
@@ -56,6 +54,7 @@ class WorkflowState(BaseModel):
     error: Optional[str] = None
     results: Optional[str] = None
     last_data_object_upa: Optional[str] = None
+    validation_reasoning: Optional[str] = ""
```

Comment:
This was being included in results from the validator node, so I figured it should be here. Apparently pydantic doesn't seem to care if you add extra stuff in a `model_copy` function...